test/presets/netlify.test.ts

```diff
@@ -214,3 +215,109 @@ describe("nitro:preset:netlify", async () => {
     });
   });
 });
+
+describe("nitro:preset:netlify-edge", async () => {
+  const publicDir = resolve(getPresetTmpDir("netlify-edge"), "dist");
+  const ctx = await setupTest("netlify-edge", {
+    config: {
+      output: {
+        publicDir,
+      },
+      netlify: {
+        images: {
+          remote_images: ["https://example.com/.*"],
+        },
+      },
+    },
+  });
+
+  testNitro(
+    ctx,
+    async () => {
+      const { default: handler } = (await import(
+        resolve(ctx.rootDir, ".netlify/edge-functions/server/server.js")
+      )) as {
+        default: (req: Request, _ctx: EdgeFunctionContext) => Promise<Response>;
+      };
+      return async ({ url: rawRelativeUrl, headers, method, body }) => {
+        // creating new URL object to parse query easier
+        const url = new URL(`https://example.com${rawRelativeUrl}`);
```

Comment:
Can we use newURL(rawRelativeUrl, headers['host'])? (right now we don't have host tests I guess but it might fail later and need the right fix.